api/types/registry.go

```diff
@@ -0,0 +1,10 @@
+package types
+
+// RegistrySettings encapsulates all registry-related configuration
+type RegistrySettings struct {
+	HasLocalRegistry    bool   // whether a local registry is available
```

Comment:
since this type is exported can you move these comments above the line so they are in the standard godoc format?

```suggestion
	// HasLocalRegistry is true when a local registry is available
	HasLocalRegistry    bool
```